osu.Game/Overlays/Direct/PanelDownloadButton.cs

```diff
@@ -16,6 +18,8 @@ public class PanelDownloadButton : BeatmapDownloadTrackingComposite
 
         private readonly bool noVideo;
 
+        public Bindable<BeatmapInfo> CurrentBeatmap = new Bindable<BeatmapInfo>();
```

Comment:
This is usually done as
```suggestion
        public IBindable<BeatmapInfo> CurrentBeatmap { get; } = new Bindable<BeatmapInfo>();
```
The interface prevents external sets to `.Value`, and the lack of setter makes it impossible to replace the bindable (but does still allow binding via `.BindTo()`/`BindTarget`). Less foot-shooty if there are `BindValueChanged` subscriptions (I imagine plain overwrite could then result in a memory leak).

Reply:
So ``BintTo()`` is same as setting the value to ``GetBoundCopy()``?

Reply:
Strictly the same - no. But as can be seen from [the implementation](https://github.com/ppy/osu-framework/blob/42202ea1ab1d55a930a9ab3a3638561b62ba1d4a/osu.Framework/Bindables/Bindable.cs#L389-L400) both can work in this scenario and the flow of passing `BindTarget` is something I've seen more.

Generally from what I've seen the usage pattern with bindables is to hold a single private (readonly even, if permissible) bindable reference in a field and keep using it throughout the component's lifetime instead of reallocating/reassigning as that is easy to mess up.